app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageViewModel.kt

```diff
@@ -129,27 +125,18 @@ class NewMessageViewModel : ViewModel() {
         autoSaveJob?.cancel()
         autoSaveJob = viewModelScope.launch(Dispatchers.IO) {
             delay(DELAY_BEFORE_AUTO_SAVING_DRAFT)
-            val mailboxUuid = MainViewModel.currentMailboxObjectId.value?.let(MailboxController::getMailbox)?.uuid!!
-
-            RealmDatabase.mailboxContent().writeBlocking {
-                saveDraftToLocal(currentDraftLocalUuid!!, DraftAction.SAVE)
-
-                val draft = DraftController.getDraft(currentDraftLocalUuid!!, this) ?: return@writeBlocking
-                DraftController.executeDraftAction(draft, mailboxUuid, this)
-            }
+            saveDraftToLocal(currentDraftLocalUuid!!, DraftAction.SAVE)
         }
     }
 
     fun saveToLocalAndFinish(action: DraftAction) = viewModelScope.launch(Dispatchers.IO) {
         val draftLocalUuid = currentDraftLocalUuid ?: return@launch
```

Comment:
Can `currentDraftLocalUuid` really be null?

Reply:
No it cannot.

It's fixed.